docs/helpers/aria.ts

```diff
@@ -0,0 +1,24 @@
+import { ArgType } from './story';
+
+const hidden: ArgType<unknown> = {
+  control: 'boolean',
+  description: [
+    'Indicates whether the element is exposed to an accessibility API.',
+    'Either this or `aria-label` required for accessibility',
```

Comment:
[Minor] Should this (and the one below) have a full stop?

Reply:
Yep, nice catch.